tests/queries/0_stateless/02931_rewrite_sum_column_and_constant.sql

```diff
@@ -0,0 +1,221 @@
+Select sum(number + 1) from numbers(10);
```

Comment:
Please enable echo so it's easier to review these new files:

```
-- { echoOn }
-- { echoOff }
```

Otherwise reviewing is 100x harder

Reply:
Done